yew-macro/src/html_tree/html_component.rs

```diff
@@ -117,7 +117,7 @@ impl ToTokens for HtmlComponent {
 
         let init_props = match &props.prop_type {
             PropType::List(list_props) => {
-                let set_props = list_props.iter().map(|HtmlProp { label, value }| {
+                let set_props = list_props.iter().map(|HtmlProp { label, question_mark: _, value }| {
```

Comment:
```suggestion
                let set_props = list_props.iter().map(|HtmlProp { label, value, .. }| {
```

Reply:
I can do this if you want, but a downside to this approach is that it will swallow any additional fields without letting you know. Which I would find frustrating, since I like doing compiler-guided refactoring (make a change, then compile and use compiler errors to guide you to where you need to make a change.) In fact, I found that I had to make a change in this exact location exactly *because* it didn't already use `..`.

(Yes, poor example of usefulness, since this location actually doesn't need that field, but the point stands.)

Reply:
I absolutely agree with you this can be a useful to find code that needs to be updated but it's not really necessary in this case. This code only needs the `label` and `value` fields and in order for it to break there would need to be a substantial change to the `HtmlProp` struct. 

But I don't mind being cautious so feel free to decide based on your preference.